tests/formatting-long-subcommand.err.txt

```diff
@@ -1,7 +1,8 @@
-Usage: formatting-long-subcommand hello-very-long-sub 
-         [TARGET...] [--first-flag] 
-         [--second-flag] [--third-flag] 
-         [--fourth-flag]
+Usage: formatting-long-subcommand hello-very-long-sub [TARGET...]
+                                                      [--first-flag]
+                                                      [--second-flag]
+                                                      [--third-flag]
+                                                      [--fourth-flag]
```

Comment:
The point of this test was to show the behaviour of the new hang feature, so if you could delete the second commit I'd be much obliged.

Reply:
@HuwCampbell> I deleted the second commit and rebased on latest `master` :+1:

Locally on my machine, all tests pass.